src/renderers/webvr/WebVRManager.js

```diff
@@ -121,6 +122,18 @@ function WebVRManager( renderer ) {
 
 		poseObject.updateMatrixWorld();
 
+		var stageParameters = device.stageParameters;
+		
+		if (stageParameters) {
+		  standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
+		} else {
+		  standingMatrix.makeTranslation(0, scope.userHeight, 0);
+		}
+
+		standingMatrixInverse.getInverse( standingMatrix );
+		poseObject.matrixWorld.multiply( standingMatrix );
+		camera.matrixWorldInverse.multiply( standingMatrixInverse );
```

Comment:
Why `camera.matrixWorldInverse` instead of `poseObject.matrixWorldInverse` here?

Reply:
`matrixWorldInverse` is used to calculate the camera projection if I remember correctly and Object3Ds like `poseObject` doed not have a matrixWorldInverse. I never fully understood this part. I’m ready to be educated about it 🙂

Reply:
> `matrixWorldInverse` is used to calculate the camera projection

Hmm... Not really. Did you added that line because if was needed in a specific situation? Otherwise if it's not needed I don't think we should add it. If that line is not needed, we can also remove all the `standingMatrixInverse` code.

Reply:
I did not add that code myself. It was there prior to the standingMatrix removal. I just ported the old logic. It works with it, I have not tried without. Happy to remove.
https://github.com/mrdoob/three.js/commit/a3f33be18c48124c3d5a0faf366685bde72587e6#diff-fdee169d721b24ed863f98ee8b9f67f4L146

Reply:
Right, but now we are using `poseObject` and that old code doesn't make sense. Please remove the code if it's not needed. Lets try to have things under control.